sql/regexp/text/regexp-in/empty-element/only/seqscan.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE memos (
+  content text
+);
+
+INSERT INTO memos VALUES ('PostgreSQL is a RDBMS');
+INSERT INTO memos VALUES ('Groonga is fast full text search engine');
+INSERT INTO memos VALUES ('PGroonga is a PostgreSQL extension that uses Groonga');
```

Comment:
How about removing these two records because I think one record is enough for this test pattern?